pkg/utils/util_test.go

```diff
@@ -142,3 +144,32 @@ func (t *testUtilsSuite) TestCompareBinlogPos(c *C) {
 	}
 
 }
+
+func (t *testUtilsSuite) TestWaitSomething(c *C) {
+	var (
+		backoff  = 10
+		waitTime = 10 * time.Millisecond
+		count    = 0
+	)
+
+	// wait fail
+	f1 := func() bool {
+		count++
+		return false
+	}
+	c.Assert(WaitSomething(backoff, waitTime, f1), IsFalse)
+	c.Assert(count, Equals, backoff)
+
+	count = 0 // reset
+	// wait success
+	f2 := func() bool {
+		count++
+		if count >= 5 {
```

Comment:
`return count >= 5`

Reply:
done in https://github.com/pingcap/dm/pull/171/commits/f9b1663d8a755052f0307248dd1f9a62ead84dc5.